hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerManagerImpl.java

```diff
@@ -305,6 +347,19 @@ public Set<ContainerReplica> getContainerReplicas(final ContainerID id)
         .orElseThrow(() -> new ContainerNotFoundException("ID " + id));
   }
 
+  @Override
```

Comment:
This should probably be a utility method rather than part of the interface, as it just needs a Set<ContainerReplica> to iterate over.

Reply:
sure , make sense , will fix